test/example_data.dart

```diff
@@ -125,7 +125,7 @@ User user({
   _checkPositive(userId, 'user ID');
   return User(
     userId: userId ?? _nextUserId(),
-    deliveryEmail: effectiveDeliveryEmail,
+    deliveryEmail: deliveryEmail,
```

Comment:
In general this sort of change should be in a separate commit before the main commit. That way it can get its own commit message clearly explaining why it's a good change. (Here if we were making this change we'd want some other changes in the surrounding code to match: it doesn't make sense to still have a variable called "effective delivery email" if it's not what actually gets used as the delivery email.)

In this case, though, the problem this is solving should be resolved by #1087, so you can rebase your work on top of that PR and drop this change.

Reply:
Alright.